contracts/access/roles/CapperRole.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.24;
+
+import "../Roles.sol";
+
+
+contract CapperRole {
+  using Roles for Roles.Role;
+
+  Roles.Role private cappers;
+
+  constructor() public {
+    cappers.add(msg.sender);
+  }
+
+  modifier onlyCapper() {
+    require(isCapper(msg.sender));
+    _;
+  }
+
+  function isCapper(address _account) public view returns (bool) {
+    return cappers.has(_account);
+  }
+
+  function addCapper(address _account) public onlyCapper {
```

Comment:
that's a good point; a lot of the api surface for these runs into the same issues with the problem with RBAC and it's that you can't define a public api surface by default because then changing it is super hard. and you don't know what sort of process a project might want to be able to use to be able to change the cap, for example.

Reply:
Our rationale was that someone who has a role is already able to perform actions using that role on behalf of other people. In fact, with ownership, it was trivial to transfer ownership to a forwarder contract that would let any set of accounts use the ownership. But this could also be done informally, as an off-chain favor.

Since this was already possible, we decided to make it part of the API. Otherwise it just feels like obscuring what is really possible.

Reply:
Our rationale was that someone who has a role is already able to perform actions using that role on behalf of other people. In fact, with ownership, it was trivial to transfer ownership to a forwarder contract that would let any set of accounts use the ownership. But this could also be done informally, as an off-chain favor.

Since this was already possible, we decided to make it part of the API. Otherwise it just feels like obscuring what is really possible.